src/context/admin-context-provider/AdminContextProvider.test.tsx

```
@@ -1,64 +1,149 @@
-      </AdminContextProvider>
-    );
+    beforeEach(() => {
+        jest.clearAllMocks();
 
-    const header = container.querySelector('.header');
-    const token = container.querySelector('.token');
+        loginRequestMock.mockResolvedValue('login_token');
+        isValidMock.mockImplementation((t: string) => t === 'initial_token' || t === 'login_token');
+        CreateAdminClientMock.mockReturnValue({ marker: 'FAKE_CLIENT' });
+        (API_ROUTES as any).BASE = '/base';
+    });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)